Controllers/ViewController.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  AdoptPet.swift
+//  Paws4Me
+//
+//  Created by Kirsty-lee Walker 20/02/2022
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var petTable: UITableView!
+    // Creates an pet object for users to reference the data
```

Comment:
So ideally we don't want to commit comments 😄

Reply:
Noted. 👍🏻